node/service/src/lib.rs

```diff
@@ -497,14 +506,20 @@ where
 				command_sink: None,
 				frontier_backend: frontier_backend.clone(),
 				backend: backend.clone(),
-				debug_requester: spawned_requesters.debug.clone(),
-				trace_filter_requester: spawned_requesters.trace.clone(),
-				trace_filter_max_count: rpc_config.ethapi_trace_max_count,
 				max_past_logs,
 				transaction_converter,
 			};
 
-			rpc::create_full(deps, subscription_task_executor.clone())
+			let mut io = rpc::create_full(deps, subscription_task_executor.clone());
+			if cfg!(feature = "evm-tracing") {
```

Comment:
If you replace by #[cfg(feature = "evm-tracing")], the followed instruction will only compiled if the feature is enabled. It's better than cfg! macro.